examples/ASCollectionView/Sample/ViewController.m

```diff
@@ -92,6 +96,8 @@ - (void)viewDidLoad
 
 - (void)reloadTapped
 {
+  // This method is deprecated because we reccommend using ASCollectionNode instead of ASCollectionView.
+  // This functionality & example project remains for users who insist on using ASCollectionView.
```

Comment:
@hannahmbanana Since `ASCollectionView` outside of the node isn't going to be supported for long, this sample should either be changed majorly or discarded entirely.